github/event_types.go

```diff
@@ -345,6 +345,42 @@ type PingEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// ProjectEvent is triggered when project is created, modified or deleted.
+// the webhook event name is "project".
+//
+// GitHub docs https://developer.github.com/v3/activity/events/types/#projectevent
+type ProjectEvent struct {
+	Action        *string       `json:"action,omitempty"`
+	Project       *Project      `json:"project,omitempty"`
+	Repository    *Repository   `json:"repository,omitempty"`
+	Organizaition *Organization `json:"organization,omitempty"`
+	Sender        *User         `json:"sender,omitempty"`
```

Comment:
You should add an extra `Installation` field here.

```
Installation *Installation `json:"installation,omitempty"`
```

Based on what https://developer.github.com/webhooks/#payloads says, all events may have this extra optional field:

> In addition to the fields documented for each event, webhook payloads include the user who performed the event (`sender`) as well as the organization (`organization`) and/or repository (`repository`) which the event occurred on, and for an Integration's webhook may include the installation (`installation`) which an event relates to.

This is related to #505. /cc @bradleyfalzon